opendm/config.py

```diff
@@ -17,28 +31,34 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 def config():
     parser.add_argument('--images', '-i',
-                        metavar='<string>',
+                        metavar='<path>',
                         help='Path to input images'),
 
     parser.add_argument('--project-path',
-                        metavar='<string>',
-                        help='Path to the project to process')
+                        metavar='<path>',
+                        help='Path to the project folder',
+                        default=defaultSettings['project_path'])
+
+    parser.add_argument('name',
+                        metavar='<project name>',
+                        type=alphanumeric_string,
+                        help='Name of Project (i.e subdirectory of projects folder)')
 
     parser.add_argument('--resize-to',  # currently doesn't support 'orig'
                         metavar='<integer>',
-                        default=2400,
+                        default=defaultSettings['resize_to'],
```

Comment:
Yes you wouldn't want to to have defaults in two places. I guess I'm failing to see the use case for having a defaults file.

If parameters are hard coded in config.py, those are the defaults. If a user wants to change the defaults, it can do so by creating a config file.

I guess my question is, what is the use case for having a defaults config file that cannot be achieved with a normal config file?